lib/archethic/reward.ex

```diff
@@ -242,4 +248,32 @@ defmodule Archethic.Reward do
       |> DateTime.from_naive!("Etc/UTC")
     end
   end
+
+  @key :reward_gen_addr
+  @spec persist_gen_addr() :: :ok
+  def persist_gen_addr() do
+    try do
```

Comment:
In which case something will throw ?

Reply:
When first time node boots up, it will not have reward chain, post bootstrap. db can return nil, causing to crash

Reply:
But you are already catching the nil case